pages/find-schedule/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { CardType } from "@findSchedule/components/Card";
+import axios from "axios";
+import Page from "findSchedule/components/Page";
+import { GetServerSideProps } from "next";
+import React from "react";
+
+interface FindPageProps {
+  defaultCardList: CardType[];
+}
+
+const index = ({ defaultCardList }: FindPageProps) => {
+  return (
+    <div className="flex justify-center mt-[78px]">
+      <Page defaultCardList={defaultCardList} />
+    </div>
+  );
+};
+
+export default index;
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  try {
+    const res = await axios.get(
```

Comment:
```suggestion
    const res = await axios.get<CardType[]>(
```
[Lv1] axios 함수 제너릭으로 타입을 적어주면 `res.data`의 타입으로 사용할 수 있고 아래에 타입명시 (`as CardType[]`) 을 뺄수있습니다!